Ownership speaker notes (#172)
google/comprehensive-rust: src/ownership/move-semantics.md

* When `s1` goes out of scope, nothing happens: it has no ownership.
* When `s2` goes out of scope, the string data is freed.
* There is always _exactly_ one variable binding which owns a value.

<details>

> * Mention that this is the opposite of the defaults in C++, which copies by value unless you use std::move (and the move constructor is defined!)

Reviewer: ```suggestion
* Mention that this is the opposite of the defaults in C++, which copies by value unless you use `std::move` (and the move constructor is defined!).
```

> * In Rust, you clones are explicit (by using `clone`).

Reviewer 2: s/you clones/clones/

> * In rust, you have to be explicity to clone/copy values, this is by using `clone`.

Reviewer: ```suggestion
* In Rust, you clones are explicit (by using `clone`).
```
